WebContent/PolicyResult.jsp

```diff
@@ -0,0 +1,87 @@
+<%@ page language="java" contentType="text/html; charset=ISO-8859-1"
```

Comment:
We have conflict in this file. Do you need to push it? because Debrupa already pushed it